src/java.compiler/share/classes/javax/lang/model/element/PackageElement.java

```diff
@@ -32,6 +32,33 @@
  * Represents a package program element.  Provides access to information
  * about the package and its members.
  *
+ * @apiNote
+
+ * The represented package may have an explicit {@linkplain
+ * javax.lang.model.util.Elements#getFileObjectOf(Element) reference
+ * representation} (either source code or executable output) or may be
+ * created from implicit information. The explicit and standalone
+ * source code construct for a package is typically a {@code
+ * package-info.java} file (JLS {@jls 7.4.1}).
+ * A named package without a standalone {@code package-info.java} file
+ * can be declared by the package declarations in a {@linkplain
+ * NestingKind#TOP_LEVEL top-level} class or interface.
```

Comment:
Grammar pedantry:  
1. There are not "package declarations" (plural) in a (singular) top-level class or interface.
2. I'd also quibble whether the package declaration is "in" a top-level class or interface. Maybe you can rephrase using "of" or "for".

Reply:
Will fix point 1.

JLS contains the sentence: "A package declaration in an ordinary compilation unit specifies the name of the package to which the compilation unit belongs."

So I'll change to "A named package without ... can be declared in the package declaration of a top-level class or interface."

Thanks.